src/core_ondisk.md

```diff
@@ -69,6 +69,15 @@ The EROFS superblock is laid out as follows in [`struct erofs_super_block`](http
 | 0x68   |   __u8 | xattr_filter_reserved | Always 0 for reserved use                                                                          |
 | 0x69   |   __u8 | reserved[23]          | Reserved                                                                                           |
 
+The CRC32-C checksum is calculated from the first byte of the Superblock
+(offset `0x400`) to the end of the physical block that contains it, where the
+block size is defined by blkszbits. For block sizes larger than 1024 bytes,
+the first 1 KB of the block is skipped, because the Superblock itself starts
+at offset `0x400` and does not cover the initial padding area of the block.
+While the conventional CRC32-C algorithm performs a final XOR with `0xffffffff`,
```

Comment:
where does this come from? can you share a code example of the convertional (`standard`) implementation?
the trivial erofs sb chksum implemenetation is at: https://git.kernel.org/pub/scm/linux/kernel/git/xiang/erofs-utils.git/tree/lib/super.c#n287

Reply:
Sorry, maybe I shouldn't say "standard" because I found there's no single standard has enough authority.
I found the checksum is not same as java.util.zip.CRC32C, more specifically, it's inverted.
After an investigation, I found the erofs use the crc32c function in Linux kernel. The difference is the Java implementation do a final XOR operation on the result, but the Linux implementation don't. 
The Java implementation is following the RFC 3720. I don't know which spec the Linux followed. But I think we'd better add some notice for someone who doesn't familiar with the CRC algorithm. Just let them consider do an bitwise inverte when the checksum is not match.